core/discovery/worker.go

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (C) 2020 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package discovery
+
+import (
+	"fmt"
+)
+
+// Worker continuously runs discovery process in node's background.
+type Worker interface {
+	Start() error
+	Stop()
+}
+
+type workerComposite struct {
+	workers []Worker
+}
```

Comment:
can this be just workers?
```suggestion
type workers []Worker
```